### DISCUSSION TITLE
Host de tu pagina mediante GitHub

### DISCUSSION THREAD

**Discussion by PaulHRios** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#201] [Show and tell]
Que tal gente, estuve probando algunas opciones para hacer host a mí página y con GitHub es de las maneras más fáciles que encontré. Aquí está el tutorial que vi, realmente es muy sencillo, se sube tu archivo HTML a un repositorio, activas una opción y automáticamente se genera un link para tu página.

https://www.youtube.com/watch?v=un5WjcP027Q&t=574s&ab_channel=AlexCGDesign

**Comment by alanmvh** [comment]
Muchas gracias por tu ayuda !!! Aun no he iniciado la practica 2 pero es bueno tener recursos como el que proporcionas :)

**Reply by villegas97** [reply]
Amigo, la entrega es el 4 de marzo.

**Reply by alanmvh** [reply]
Por trabajo y responsabilidades no pude adelantar mucho pero ya termine la practica 1, aun quedan 24 grandiosas horas para esa practica de html :D

**Reply by villegas97** [reply]
Te entiendo, andamos en las mismas, yo hoy termino las practicas de la primer semana (esa documentación de requerimientos me pone de nervios, es aburrido hacerlas jaja).

**Comment by JuanSalvador73** [comment]
Muchas gracias por la info. :D

**Comment by dalexez** [comment]
Hola! de antemano muchísimas gracias @PaulHRios  por el aporte, solo me quedo una duda quizá me puedan ayudar a entender.
En la parte de Source  que diferencia hay entre elegir / (root)  o   elegir  /docs  ?  
![settings](https://user-images.githubusercontent.com/62618610/156660571-47d200d4-7cc3-41a2-b2a8-343e068239a6.png)

**Reply by OmarVenturaP** [reply]
Root tengo entendido que es la carpeta principal que te habilita Github, ya Docs te crea una carpeta alterna y allí debes ingresar tus archivos. Dependiendo donde quieras tener tus datos de raíz de página.
